instantsearch/src/commonMain/kotlin/com/algolia/instantsearch/helper/filter/facet/dynamic/AttributedFacets.kt

```diff
@@ -0,0 +1,18 @@
+package com.algolia.instantsearch.helper.filter.facet.dynamic
+
+import com.algolia.search.model.Attribute
+import com.algolia.search.model.search.Facet
+
+/**
+ * List of ordered facets with their attributes.
```

Comment:
```suggestion
 * List of ordered facets with their attribute.
```
I had this typo in my code as well and just fixed it 🙂